src/Journalist.EventStore/Connection/IEventStoreConnection.cs

```diff
@@ -19,6 +20,8 @@ public interface IEventStoreConnection
 
         Task<IEventStreamConsumer> CreateStreamConsumerAsync(string streamName, string consumerName);
 
+		IFailedNotificationsHub FailedNotificationsHub { get; }
```

Comment:
It is quite handy to reuse event store configuration to build this failed notification hub and then use it in backend clients and utilities to download and reprocess failed messages.